pkg/operator/controller/nodeobservability/daemonset.go

```diff
@@ -153,11 +174,21 @@ func (r *NodeObservabilityReconciler) desiredDaemonSet(nodeObs *v1alpha1.NodeObs
 							Name: socketName,
 							VolumeSource: corev1.VolumeSource{
 								HostPath: &corev1.HostPathVolumeSource{
-									Path: path,
+									Path: socketPath,
 									Type: &vst,
 								},
 							},
 						},
+						{
+							Name: kbltCAName,
+							VolumeSource: corev1.VolumeSource{
+								ConfigMap: &corev1.ConfigMapVolumeSource{
```

Comment:
The container which will consume this CA, will it be watching for the changes in the file system to catch up with the updates of the CA (in case of a rotation)?